packages/wonder-blocks-button/components/button.md

```diff
@@ -83,181 +131,325 @@ const styles = StyleSheet.create({
         backgroundColor: Color.darkBlue,
         padding: 10,
     },
-    sideMargins: {
+    button: {
         marginRight: 10,
     }
 });
 
 <View style={styles.row}>
     <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
-    >Label</Button>
+    >
+        Primary
+    </Button>
     <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="secondary"
-    >Label</Button>
+    >
+        Secondary
+    </Button>
     <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="tertiary"
-    >Label</Button>
+    >
+        Tertiary
+    </Button>
     <Button
-        disabled={true}
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
-    >Disabled</Button>
-    <Button
         disabled={true}
+    >
+        Primary
+    </Button>
+    <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="secondary"
-    >Disabled</Button>
-    <Button
         disabled={true}
+    >
+        Secondary
+    </Button>
+    <Button
         light={true}
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="tertiary"
-    >Disabled</Button>
+        disabled={true}
+    >
+        Tertiary
+    </Button>
 </View>
 ```
 
-Button examples, `style` specified (Supports width, position, margin, and flex styles):
+Buttons have a `size` that's either `"medium"` (default) or `"small"`.
```

Comment:
Would be good to clarify that the following buttons are all "small".

Reply:
Will do.  I should note that this is a breaking change.  Should I bump button to `v2`?

Reply:
Yep, I think so!